servers/xr/xr_interface.cpp

```diff
@@ -52,9 +54,16 @@ void XRInterface::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "interface_is_primary"), "set_is_primary", "is_primary");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "interface_is_initialized"), "set_is_initialized", "is_initialized");
 
-	// we don't have any properties specific to VR yet....
+	// methods and properties specific to VR...
+	ClassDB::bind_method(D_METHOD("supports_play_area_mode", "mode"), &XRInterface::supports_play_area_mode);
+	ClassDB::bind_method(D_METHOD("get_play_area_mode"), &XRInterface::get_play_area_mode);
+	ClassDB::bind_method(D_METHOD("set_play_area_mode", "mode"), &XRInterface::set_play_area_mode);
+	ClassDB::bind_method(D_METHOD("get_play_area"), &XRInterface::get_play_area);
+
+	ADD_GROUP("VR", "vr_");
```

Comment:
I think this should be `XR` instead of `VR`, as I'd imagine full AR headsets (when they're available) would leverage similar play area/tracking terminology.

Reply:
Yup agree

Reply:
Does it really make sense to have an `XR` group in an `XRInterface` node though? It seems the `VR`/`AR` groups were used to distinguish VR and AR-specific properties, but if they all apply to XR, the distinction is not needed.